src/field-accessor.ts

```diff
@@ -95,7 +95,17 @@ export class FieldAccessor<R, V> extends AccessorBase implements IAccessor {
     if (this.field.converter.emptyImpossible) {
       return false;
     }
-    return this.raw === this.field.converter.emptyRaw;
+    const raw = this.raw;
+    const emptyRaw = this.field.converter.emptyRaw;
+    const converterName = (this.field._converter as any).name;
+    if (
+      converterName === "stringArray" &&
```

Comment:
I think we can safely execute this when both `raw` and `emptyRaw` are arrays, so no need for the extra converter name check :)